### ISSUE TITLE
Add options for emails

### ISSUE THREAD

**Issue by emmanuelgeoffray** [JohMun/vue-tags-input#68]
Hi,

First of all, thank you for this component. It works great and I love using it.

I acutally use it for user to enter multiple emails in an input, like you would do in an email client.
It works perfectly well, the only issue I have, is that on iOS, the keyboard is not the one for email.
To enable this, it requires a `type: 'email'` for each input, and I can't succeed to make vue-tags-input use that property. It seems to be hardcoded to be a `type: 'text'`
https://github.com/JohMun/vue-tags-input/blob/master/vue-tags-input/tag-input.vue#L8

Or am I missing something?

Thank you,
Emmanuel

**Comment by JohMun** [comment]
Yes, currently there is no prop/option/attr to change the input type. But there should be one, because what you described is a use case. I add the label 'enhancement' and try to provide a solution in the next version.

**Comment by emmanuelgeoffray** [comment]
Great! Thanks for your reply @JohMun 
Let me know if I can be of any help.
Best!
e

**Comment by gabrielstuff** [comment]
Hello @JohMun, do you have recommandations on how you would like to see this feature integrated ? Would be nice to see it live soon. Thanks !

**Comment by Muhamad-Talha** [comment]
@JohMun  Do we have the email select feature?
